docs/User/ProjectRelated/Changelog.md

```diff
@@ -14,11 +14,48 @@
     writeWithBufferOf -> writeWith
     writeChunksWithBufferOf -> writeChunksWith
 
+* Following APIs have been renamed in Streamly.Data.Fold module:
+    head -> one
+
+* Moved Streamly.Data.Array from streamly to streamly-core package.
+* Moved Streamly.Data.Unfold from streamly to streamly-core package.
+* Moved Streamly.Console.Stdio from streamly to streamly-core package.
+* Moved Streamly.FileSystem.Handle from streamly to streamly-core package.
+* Moved Streamly.Unicode.Stream from streamly to streamly-core package.
```

Comment:
Use DRY principle. Can write: "Following modules and corresponding internal modules have been moved to streamly-core package:", and then list the modules.